crm_phonecall/views/crm_phonecall_view.xml

```diff
@@ -0,0 +1,464 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="crm_phonecall_categ_action" model="ir.actions.act_window">
+        <field name="name">Phonecall Categories</field>
+        <field name="res_model">crm.tag</field>
+        <field name="view_id" ref="sales_team.sales_team_crm_tag_view_tree" />
+        <field name="domain">[]</field>
+        <field eval="{'object_id': ref('model_crm_phonecall')}" name="context" />
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_create">Click to add a new category.</p>
+            <p>
+                Create specific phone call categories to better define the type of
+            calls tracked in the system.
+            </p>
+        </field>
+    </record>
+    <menuitem
+        id="menu_crm_config_phonecall"
+        name="Phone Calls"
+        parent="crm.crm_menu_config"
+        sequence="19"
+    />
+    <menuitem
+        action="crm_phonecall_categ_action"
+        id="menu_crm_case_phonecall-act"
+        name="Categories"
+        parent="menu_crm_config_phonecall"
+        sequence="0"
+    />
+    <record id="crm_case_phone_tree_view" model="ir.ui.view">
+        <field name="name">CRM - Phone Calls Tree</field>
+        <field name="model">crm.phonecall</field>
+        <field name="arch" type="xml">
+            <tree
+                decoration-info="state in ('pending',)"
+                decoration-muted="state in ('cancel','done')"
+            >
+                <field name="date" />
+                <field name="name" />
+                <field name="partner_id" />
+                <button
+                    icon="fa-calendar"
+                    name="action_make_meeting"
+                    invisible="state not in ['open', 'pending']"
+                    string="Meeting"
+                    type="object"
+                />
+                <button
+                    invisible="opportunity_id or state not in ['open', 'pending']"
+                    icon="fa-hand-pointer-o"
+                    name="action_button_convert2opportunity"
+                    string="Convert to Opportunity"
+                    type="object"
+                />
+                <field name="partner_phone" />
+                <field name="user_id" />
+                <field invisible="1" name="tag_ids" />
+                <field invisible="1" name="create_date" />
+                <field invisible="1" name="opportunity_id" />
```

Comment:
invisible -> column_invisible needs to be changed as well.
Also in other views.

Reply:
Fix it! Please review @CRogos

Reply:
Could you cleanup the commits as well?
The code changes LGTM, but did not test the addon in detail.

Reply:
Done! Can you review again please @CRogos